.github/workflows/package.yml

```
@@ -709,8 +709,23 @@ jobs:
       - name: Save SBOM
         uses: actions/upload-artifact@v4
         with:
-          name: bom.xml
-          path: bom.xml
+          name: bom.json
+          path: bom.json
+
+      - name: Install devolutions toolbox
+        uses: devolutions/actions/toolbox-install@v1
+        with:
+          github_token: ${{ secrets.DEVOLUTIONSBOT_TOKEN  }}
+
+      - name: Upload SBOM to OneDrive Releases
+        uses: devolutions/actions/onedrive-upload@v1
```

```suggestion
        uses: ./.github/workflows/onedrive-upload
```
gateway est un repo publique, ce n'est pas la même manière d'appeler les actions privées. Tu peux te fier à ce qu'on fait plus haut.